content/tutorials/Team2Tutorial.md

```diff
@@ -0,0 +1,259 @@
+---
+title: ESP32 Communicaation Over WiFi !
+date: 2024-12-06
+authors:
+  name: Anton John Del Mar
+  name: Daniel Chen
+  name: Peter Quawas
+---
+
+<img src = "image-5.png" width = "590" height = "300" />
```

Comment:
Markdown is not HTML, to embed images, use the following syntax:

```
![](image_url)
```

Reply:
In the most recent update I have fixed this issue.